index.html

```diff
@@ -82,7 +102,7 @@ <h1 class="project-heading">Projects</h1>
           class="project-image"
           alt="laptop"
         />
-        <div class="project-info">
+        <div class="project-info cont2">
           <p class="project-info-name">Project name goes here</p>
           <ul class="project-languages">
             <li>HTML/CSS</li>
```

Comment:
- [ ] Your second Project container seems to have introduced a bug and it's causing the info section to be out of place as you can see in the image below

> Yours
![image](https://user-images.githubusercontent.com/90556221/166120637-1dd4458f-d02b-4a45-a52f-be7a487e5f84.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/166120607-0c5470db-59cf-4fa0-96dd-60b370add732.png)

Kindly fix this visual bug ✌️ . The rest of the design seems almost perfect 👍